en/docs/references/data-mapping.md

```diff
@@ -1,49 +1,48 @@
 # Data Mapping
 
 You can use a data mapping statement in Choreo to create variables by visually mapping one or more variables.  The most common use of data mapping in Choreo is to map data from an API call response to JSON format. 
-
 Follow this procedure if you want to add a data mapping statement to your integration or service:
 
 1. Go to the **Statements** tab and select **Data Mapping**. You can see the **Input** and **Output** panes.
 
      ![Data mapping input output windows](../assets/img/references/datamapping/input-output-windows.png){.cInlineImage-full}
    
-2. To add an input variable, follow this procedure:
+2. To add an input variable:
 
     1. In the **Input** pane, click **Add Variable**.
     2. Select a required input type from the **Select Input Type** drop-down list.
     3. Click **Add**.
 
 3. To add an output variable, you can either select an existing variable or create a new variable via the **Output** pane. 
 
-    - To create a new variable, follow this procedure in the **Output** pane: 
+    - To create a new variable in the **Output** pane: 
 
         !!! tip
-            If you want to create a new output variable that is optional, click the **Create Optional Variable** toggle to enable it.
+            If you want to create a new optional output variable, click the **Create Optional Variable** toggle to enable it.
             
          1. Enter a value as the **Variable Name**.
          2. Select a variable type from the **Select Variable Type** drop-down list.
          3. Click **Save**.
 
-    - To use an existing variable, follow this procedure in the **Output** pane:         
+    - To use an existing variable in the **Output** pane:         
          1. Click the **Use Existing Variable** toggle to enable it.
          2. Select a variable from the **Select Output Variable** drop-down list.
          3. Click **Save**.
   
 
-   Once you add the input and output variables, the connectors in the middle are displayed.
+   Once you add the input and output variables, the connectors in the middle are displayed as the following figure shows:
 
    ![Data mapping connections](../assets/img/references/datamapping/data-mapping-connections.png){.cInlineImage-full}
    
-   The **Input** pane displays constant values (e.g. `0`) and the input variables.
+   The **Input** pane displays constant values (for example, `0`) and the input variables.
```

Comment:
what is the reason to not use e.g.?

Reply:
The style guide recommends to avoid abbreviations such as i.e., and e.g., [1].
[1] https://developers.google.com/style/abbreviations?hl=en#dont-use